packages/shylu/shylu_dd/frosch/test/TestInterfaceSets/main.cpp

```diff
@@ -65,16 +65,20 @@ typedef unsigned UN;
 typedef double SC;
 typedef int LO;
 typedef int GO;
+#ifdef KOKKOS_ENABLE_SERIAL
 typedef Kokkos::Compat::KokkosSerialWrapperNode EpetraNode;
+#else
+typedef Kokkos::Compat::KokkosOpenMPWrapperNode EpetraNode;
```

Comment:
Needs `KOKKOS_ENABLE_OPENMP` protection here.  For example, it's possible to build with OpenMP off, Serial on, and Cuda on.